plugins/doc_fragments/auth.py

```diff
@@ -27,6 +27,7 @@ class ModuleDocFragment(object):
           - jwt
           - cert
           - none
+          - kubernetes
```

Comment:
Small thing and I don't know if it even makes sense, but let's move this above `none`, for some reason I like `none` being at the end.

Also please add a line after line 19, to describe when this method was added (see the other options there as well).